setup.py

```diff
@@ -24,19 +24,16 @@
 from setuptools import find_packages, setup
 
 PACKAGE_NAME = "aea"
+here = os.path.abspath(os.path.dirname(__file__))
 
 
 def get_all_extras() -> Dict:
 
-    cosmos_ledger_deps = ["ecdsa==0.15", "bech32==1.2.0"]
+    cosmos_ledger_deps = [f"cosmos_crypto @ file://{here}/plugins/cosmos-crypto#egg=cosmos_crypto"]
 
-    fetch_ledger_deps = cosmos_ledger_deps
+    fetch_ledger_deps = [f"fetchai_crypto @ file://{here}/plugins/fetchai-crypto#egg=fetchai_crypto"]
 
-    ethereum_ledger_deps = [
-        "web3==5.12.0",
-        "ipfshttpclient==0.6.1",
-        "eth-account==0.5.2",
-    ]
+    ethereum_ledger_deps = [f"ethereum_crypto @ file://{here}/plugins/ethereum-crypto#egg=ethereum_crypto"]
```

Comment:
Do we still want to bundle that?

Reply:
The alternative is that we remove it entirely here and just require the user to always manually add the crypto plugins in an aea project (we could have a command for this on cli and agent files can specify directly as dependency)